third_party/opencv_ios_source.BUILD

```diff
@@ -42,6 +42,8 @@ genrule(
         --without objdetect \
         --without gapi \
         --without flann \
+        --without highgui \
+        --without videoio \
```

Comment:
These symbols from these opencv modules conflict with our library when installed with the opencv framework using CocoaPods. I am not sure why the other symbols don't conflict. 
In any case, we don't need these. So I excluded them to avoid the conflicts